### PR TITLE
Add default unicode emoji for trashcan

### PR DESCRIPTION
**Pull request by Ibrahim2750mi**
https://github.com/python-discord/site/pull/882
@wookie184

### CODE REVIEW

**Comment by wookie184** [comment]
We still want to use the custom one on the server, so this would require setting the env variable in production.

An alternative approach would be to use the `Client.debug` constant, which defaults to true, but is set to false in production. If this is true we would use the unicode default if the env var wasn't set (this wouldn't require any changes to production environment variables)

Do @python-discord/devops have a preference?

**Comment by Ibrahim2750mi** [comment]
> An alternative approach would be to use the `Client.debug` constant, which defaults to true, but is set to false in production. If this is true we would use the unicode default if the env var wasn't set (this wouldn't require any changes to production environment variables)

I don't see `Client.debug` being false anywhere, if anything, we need to set it false by ourselves. Am I right? Or does this happen outside `constants.py`

**Comment by wookie184** [comment]
The `BOT_DEBUG` environment variable is set in production, production environment variables can only be set by a member of the devops team. Since it's already set that's why I suggested using it to determine what default to use for the reaction (keeping the reaction production constant here also makes it easier to change in the future if necessary).

File: bot/constants.py [hunk 1]
```diff
@@ -131,7 +131,12 @@ class Emojis:
     cross_mark = "\u274C"
     check = "\u2611"
     envelope = "\U0001F4E8"
-    trashcan = environ.get("TRASHCAN_EMOJI", "<:trashcan:637136429717389331>")
+
+    trashcan_default = "<:trashcan:637136429717389331>"
+    if Client.debug:
+        trashcan_default = "\U0001f5d1\ufe0f"
+    trashcan = environ.get("TRASHCAN_EMOJI", trashcan_default)
```
Comment (wookie184) [inline bot/constants.py:138]:
```suggestion
    trashcan = environ.get(
        "TRASHCAN_EMOJI",
        f"\N{WASTEBASKET}" if Client.debug else "<:trashcan:637136429717389331>",
    )
```
Normally an if would probably be nicer, but here ideally we should avoid any extra variables being left over in the namespace so people don't accidentally use them in code or get confused. (alternatively we could prefix the default with a `_` and `del` it afterwards)

**Comment by wookie184** [comment]
Sorry, I forgot that isn't a f-string thing, can just remove the f, my bad 😅

**Comment by wookie184** [comment]
Hmm wait i'm not even sure if it's working without, let me try it

**Comment by Ibrahim2750mi** [comment]
> Hmm wait i'm not even sure if it's working without, let me try it

okay, I will be waiting.

**Comment by wookie184** [comment]
> > Hmm wait i'm not even sure if it's working without, let me try it
> 
> okay, I will be waiting.

Never mind, just fixing the lint error should be fine. I just had `BOT_DEBUG=false` in my `.env` for some reason.

**Review by wookie184 — APPROVED** [review]
Thanks!

**Review by shenanigansd — APPROVED** [review]
:shipit:

**Pull request merged**